sklearn/mixture/gmm.py

```diff
@@ -429,6 +450,7 @@ def fit(self, X, y=None):
         """
         # initialization step
         X = check_array(X, dtype=np.float64)
+        self.responsibilities_ = None
```

Comment:
actually, we should not store the responsibilities at all as an attribute. Instead renamed `fit` as `fit_predict` that returns the local variable `responsibilities` instead of `self` then add a new method:

```python

def fit(self, X):
    self.fit_predict(X)
    return self
```

The docstrings of those 2 methods will have to be swapped.